src/devices/Card/Ndef/GeneralePurposeByteConditions.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Iot.Device.Ndef
+{
+    /// <summary>
+    /// This is used to asses the block sector and understand the read/write conditions
+    /// </summary>
+    [Flags]
+    public enum GeneralePurposeByteConditions
```

Comment:
should this be internal?

Reply:
No as it's the way to get it properly once you have the data from the sector in a quite straight forward way